tempus_fugit_minecraft/window.py

```diff
@@ -238,6 +246,7 @@ def resume_game(self) -> None:
         """Resumes the game by restoring the game window to its original state."""
         self.paused = False
         self.set_exclusive_mouse(True)
+        glEnable(GL_LIGHTING)
```

Comment:
I feel like this and the equivilent disable function should be wrapped into the shadder, something like:
```py
def enable_lighting(self):
    glEnable(GL_LIGHTING)

def disable_lighting(self):
    glDisable(GL_LIGHTING)
```
This would also allow us to test this functionality.